blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -796,9 +808,8 @@ public void testCopyBlobReplaceMetadata() throws Exception {
          } finally {
             Closeables2.closeQuietly(is);
          }
-         // TODO: S3 overrideMetadataWith also overrides system metadata
-         // TODO: Swift does not preserve system metadata
-         //checkContentMetadata(toBlob);
+         // TODO: Swift copy object *appends* user metadata, does not overwrite
```

Comment:
@zack-shoylev Curious Swift *appends* user metadata so this test fails with:

```
Failed tests:
  CloudFilesUSBlobIntegrationLiveTest>BaseBlobIntegrationTest.testCopyBlobReplaceMetadata:813
Expecting:
 <{"key2"="value2", "key3"="value3", "key1"="value1", "key4"="value4"}>
to be equal to:
 <{"key3"="value3", "key4"="value4"}>
but was not.
```

Swift documents this behavior so this is expected.  I guess we just need to change the test expectations?